include/variables.h

```diff
@@ -2489,11 +2485,12 @@ extern Struct_80140E80* D_801F6D4C;
 extern BombersNotebook sBombersNotebook;
 extern u8 sBombersNotebookOpen;
 extern u8 sMotionBlurStatus;
+
 extern SlowlyTask D_801F6E00;
 extern s32 D_801F6FC0;
 extern StackEntry slowlyStackEntry;
```

Comment:
```suggestion
extern StackEntry sSlowlyThreadInfo;
```
To sync with other names. (This seems to be the name by the ones that are actually documented in OoT).